src/utils/utils.ts

```diff
@@ -26,7 +26,7 @@ export function getUrl(path: string, fallback: string = '') {
 }
 
 /**
- * Download given url return as ArrayBuffer
+ * Download given url return as Blob
  */
 export async function downloadUrl(url: string) {
```

Comment:
Add a new methods `downloadUrlAsArrayBuffer` or `downloadUrlAsBlob`?

Reply:
I think it isn't necessary. It's easy to get ArrayBuffer from Blob